### PR TITLE
Add VLP16 support, refactor main/secondary laser envar support

### PR DESCRIPTION
**Pull request by civerachb-cpr**
This depends on a similar PR in jackal_robot.

Main features:
- remove JACKAL_LASER_HOKUYO, replace with Dingo-style JACKAL_LASER_TYPE
- add JACKAL_LASER2 for mounting a secondary laser without using the accessory fenders
- add JACKAL_LASER_3D for mounting a primary 3D lidar

### CODE REVIEW

File: jackal_description/urdf/accessories.urdf.xacro [hunk 1]
```diff
@@ -16,21 +16,71 @@
...
-                          parent_link="$(optenv JACKAL_LASER_MOUNT front)_mount"
-                          topic="$(optenv JACKAL_LASER_TOPIC front/scan)" />
+          parent_link="$(optenv JACKAL_LASER_MOUNT front)_mount"
+          topic="$(optenv JACKAL_LASER_TOPIC front/scan)">
+          <origin xyz="$(optenv JACKAL_LASER_OFFSET 0 0 0)"
+                  rpy="$(optenv JACKAL_LASER_RPY 0 0 0)" />
+        </xacro:hokuyo_ust10_mount>
+    </xacro:if>
+  </xacro:if>
+  <xacro:if value="$(optenv JACKAL_LASER2 0)">
```
Comment (tonybaltovski) [inline jackal_description/urdf/accessories.urdf.xacro]: I would prefer a more descriptive name here, something like `JACKAL_LASER_SECONDARY`.  Thoughts?

File: jackal_description/urdf/accessories.urdf.xacro [hunk 2]
```diff
@@ -16,21 +16,71 @@
...
+
+  <!--
+    Add a 3D lidar sensor.  By default this is a Velodyne VLP16 but can be changed with the
+    JACKAL_LASER_3D_MODEL environment variable. Valid model designations are:
+    - vlp16 (default)  :: Velodyne VLP16
+  -->
+  <xacro:if value="$(optenv JACKAL_LASER_3D 0)">
+    <xacro:property name="lidar_3d_model" value="$(optenv JACKAL_LASER_3D_MODEL vlp16)" />
+    <xacro:if value="${lidar_3d_model == 'vlp16'}">
+      <xacro:VLP-16 parent="$(optenv JACKAL_LASER_3D_MOUNT front)_mount" topic="$(optenv JACKAL_LASER_3D_TOPIC front/points)">
```
Comment (tonybaltovski) [inline jackal_description/urdf/accessories.urdf.xacro]: Does this come with a mount normally or does it sit on one of the default plates?

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: jackal_description/urdf/accessories.urdf.xacro (hunk 2 above)
Comment (civerachb-cpr) [reply jackal_description/urdf/accessories.urdf.xacro]: I'll ask someone from the integration team. I believe it just mounts directly to one of the standard plates, but I'll double-check.

File: jackal_description/urdf/accessories.urdf.xacro (hunk 1 above)
Comment (civerachb-cpr) [reply jackal_description/urdf/accessories.urdf.xacro]: I considered that, but laziness and a desire to minimize the length of the variables won out in the end.  But I can go back and make the names more descriptive.

File: jackal_description/urdf/accessories.urdf.xacro (hunk 2 above)
Comment (civerachb-cpr) [reply jackal_description/urdf/accessories.urdf.xacro]: According to Jeff:
> It is typically elevated about 10cm using standoffs

I'll add an accessory for a raised mounting surface.

Comment (civerachb-cpr) [reply jackal_description/urdf/accessories.urdf.xacro]: Done.

File: jackal_description/urdf/accessories.urdf.xacro (hunk 1 above)
Comment (civerachb-cpr) [reply jackal_description/urdf/accessories.urdf.xacro]: Done. Changed in jackal_bringup too.

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**